inst/stan/functions/primarycensored.stan

```diff
@@ -77,7 +77,7 @@ real primarycensored_cdf(data real d, int dist_id, array[] real params,
   * real pwindow = 1.0;
   * real D = positive_infinity();
   * int primary_id = 1; // Uniform
-  * array[0] real primary_params = {};
+  * real primary_params = rep_array(0.0, 0);
```

Comment:
```suggestion
  * array[0] real primary_params = rep_array(0.0, 0);
```

I think this still needs the empty array. 

I think there is a wider problem with these stan examples in that they aren't really runnable. I think it might be better to write them as either R exposed stan functions or as little self contained stan models (for some future work)